crates/client/db/tests/common/mod.rs

```diff
@@ -0,0 +1,9 @@
+use dc_db::DatabaseService;
+use dp_block::chain_config::ChainConfig;
+use tempfile::TempDir;
+
+pub async fn temp_db() -> DatabaseService {
+    let temp_dir = TempDir::new().unwrap();
+    let chain_config = std::sync::Arc::new(ChainConfig::test_config());
+    DatabaseService::new(temp_dir.path(), None, false, chain_config).await.unwrap()
```

Comment:
that's one of the reason I don't like `Option` or boolean arguments, like here without context on the struct we cannot tell what that means, maybe add some rust doc to help

Reply:
add doc, done.